test/kubetest2-capi/README.md

```diff
@@ -0,0 +1,53 @@
+# Kubetest2 Cluster API Deployer
+
+This component of [kubetest2](https://github.com/kubernetes-sigs/kubetest2) is
+responsible for test cluster lifecycles for clusters deployed with Cluster API.
+
+Users can use this project along with kubetest2 testers to run [Kubernetes
+e2e tests](https://github.com/kubernetes/kubernetes/tree/master/test/e2e)
+against CAPI-backed clusters.
+
+## Installation
+```
+GO111MODULE=on go get sigs.k8s.io/cluster-api/test/kubetest2-capi@latest
+```
+
```

Comment:
i think it would be nice to add a paragraph here about building locally using `go build` in the kubetest2-capi directory. i know it won't make a big difference once this is deployed, but for developers i think it's nice to give a little extra direction on how to get started with just the code.